sdk/cognitiveservices/azure-cognitiveservices-language-textanalytics/azure/cognitiveservices/language/textanalytics/__init__.py

```diff
@@ -1,18 +1,199 @@
 # coding=utf-8
-# --------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#
-# Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is
-# regenerated.
-# --------------------------------------------------------------------------
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
 
-from .text_analytics_client import TextAnalyticsClient
-from .version import VERSION
+from typing import Any, Optional, List  # pylint: disable=unused-import
+from ._text_analytics_client import TextAnalyticsClient
+from ._response_handlers import _validate_single_input, process_single_error, process_entities_error
+from ._version import VERSION
+from ._models import (
+    LanguageInput,
+    MultiLanguageInput,
+    DetectedLanguage,
+    DocumentError,
+    Entity,
+    LinkedEntity,
+    DocumentSentiment,
+    DocumentEntities,
+    DocumentLanguage,
+    Error,
+    InnerError,
+    DocumentKeyPhrases,
+    DocumentLinkedEntities,
+    DocumentStatistics,
+    Match,
+    RequestStatistics,
+    SentenceSentiment
+)
 
-__all__ = ['TextAnalyticsClient']
+__all__ = [
+    'TextAnalyticsClient',
+    'LanguageInput',
+    'MultiLanguageInput',
+    'single_detect_language',
+    'single_recognize_entities',
+    'single_recognize_pii_entities',
+    'single_recognize_linked_entities',
+    'single_extract_key_phrases',
+    'single_analyze_sentiment',
+    'DetectedLanguage',
+    'DocumentEntities',
+    'DocumentLanguage',
+    'Entity',
+    'Error',
+    'InnerError',
+    'DocumentKeyPhrases',
+    'DocumentLinkedEntities',
+    'DocumentSentiment',
+    'DocumentStatistics',
+    'DocumentError',
+    'LinkedEntity',
+    'Match',
+    'RequestStatistics',
+    'SentenceSentiment',
+]
 
 __version__ = VERSION
 
+
+def single_detect_language(
+            endpoint,  # type: str
+            credential,  # type: str
+            text,  # type: str
+            country_hint=None,  # type: str
+            model_version=None,  # type: Optional[str]
+            **kwargs  # type: Any
+):
+    # type: (...) -> DetectedLanguage
```

Comment:
Docstrings are presumably something we'll add later...

Reply:
Added now